pocs/state/machine.py

```diff
@@ -129,8 +129,8 @@ def run(self, exit_when_done=False, run_once=False):
 
                 # If sleeping, wait until safe (or interrupt)
                 if self.state == 'sleeping':
-                    if self.is_safe() is not True:
-                        self.wait_until_safe()
+                    if self.is_safe(horizon='flat') is not True:
+                        self.wait_until_safe(horizon='flat')
```

Comment:
You're replacing one hard coded horizon for exiting the `sleeping` state (-18 degrees) with another hard coded horizon (-6 degrees). That will fix the immediate problem for Huntsman, but might not be great for Panoptes. 

Ideal solution, long term, would be to make this choice of horizon a config file thing.

Reply:
PANOPTES wants to switch to a default of taking flat-fields as well, so this should be okay. We just need to make sure to actually add in a flat field state before this is merged.